packages/runtime-common/realm.ts

```diff
@@ -318,7 +320,6 @@ export class Realm {
           owner = await this.getRealmOwnerUserId();
         }
         req.headers.set('X-Boxel-Assume-User', owner);
-        req.headers.set('X-Boxel-Disable-Module-Cache', 'true');
```

Comment:
we no longer run code in the realm so its not necessary to set this header anymore